File: tasks/leiningen/fgoc.clj
;; Compiled from tasks/leiningen/fgoc.go
(ns leiningen.fgoc (:gen-class) (:require [clojure.java.shell :as sh]))

(set! *warn-on-reflection* true)

(defn-
 fgoc
 [project & args]
 (let
  [cmd-line
   (concat
    ["java"
     "-jar"
     "bin/funcgo-compiler-0.3.0-standalone.jar"
     "src"
     "test"
     "tasks"]
    args)
   result
   (apply sh/sh cmd-line)]
  (println (result :err))
  (println (result :out))
  (if
   (= (result :exit) 0)
   (println "Compile finished")
   (println "ERROR"))))

File: tasks/leiningen/fgoc.go
package fgoc
import sh "clojure/java/shell"

func fgoc(project, args...) {
	cmdLine := [
		"java", "-jar", "bin/funcgo-compiler-0.5.0-standalone.jar",
		"src", "test", "tasks"
	]  concat  args
	result  := sh.sh  apply  cmdLine

        println(result(ERR))
        println(result(OUT))
        if result(EXIT) == 0 {
                println("Compile finished")
        } else {
                println("ERROR")
        }
}
